expertsystem/solving/__init__.py

```diff
@@ -439,42 +499,105 @@ def _encode_variable_name(variable_info: _VariableInfo, delimiter: str) -> str:
     return var_name
 
 
+def _merge_solutions_with_graph(
+    solutions: List[QuantumNumberSolution],
+    graph: StateTransitionGraph[ParticleWithSpin],
+    allowed_particles: ParticleCollection,
+) -> List[StateTransitionGraph[ParticleWithSpin]]:
+    initialized_graphs = []
+
+    logging.debug("merging solutions with graph...")
```

Comment:
Is this an informative debugging statement?

Reply:
True, I can remove this

Reply:
I will remove it in one of the next PRs as this change is to superficial to rerun the CI for...